assets/styles/pages/_list.scss

```diff
@@ -0,0 +1,31 @@
+.text-white{
+    color: white;
+}
+
+.guided-match-background{
+    background-color: #78256E;
+}
+
+.guided-match-area{
+
+    .govuk-button {
+
```

Comment:
Is this being used somewhere?

Reply:
This is to stop it overwriting .govuk-button on the rest of the site it actually uses the .govuk-button inside the .guided-match-area if you pull it down